.swift-version

```diff
@@ -1 +0,0 @@
-4.2
```

Comment:
We should really delete this line? or change to 5.0?

Reply:
I removed the file completely, I think it is not required by cocoapods any more